pkg/asset/manifests/cloudproviderconfig.go

```diff
@@ -92,6 +92,17 @@ func (cpc *CloudProviderConfig) Generate(dependencies asset.Parents) error {
 		}
 
 		cm.Data[cloudProviderConfigDataKey] = openstackmanifests.CloudProviderConfig(cloud.CloudConfig)
+
+		// Get the ca-cert-bundle key if there is a value for cacert in clouds.yaml
+		caPath := cloud.CloudConfig.CACertFile
+		var caFile string
+		if caPath != "" {
+			caFile, err = openstackmanifests.GetCaCertFromDisk(caPath)
```

Comment:
This line is functionally equivalent to:

```Go
caFile, err = ioutil.ReadFile(caPath)
```

You can safely delete the function `GetCaCertFromDisk`.